react/features/base/media/components/native/styles.js

```diff
@@ -6,6 +6,14 @@ import { ColorPalette } from '../../../styles';
  * The styles of the feature base/media.
  */
 export default StyleSheet.create({
+
+    /**
+     * Base style of the transformed video view.
+     */
+    transformedView: {
```

Comment:
I'm really confused by the naming around "video transforms". The action types and creators, the class `VideoTransform` talk about an abstraction "video transforms". But I see these `transforms` in the redux state `base/media` that are merely "transforms". And I was thinking that actually "transforms" are like a base/super and then there are "video transforms". But then there's no such mechanism. Such a confusion is preemptively addressed by our coding style: one name per abstraction. In accord with it I'd rather go with `videoTransforms` in the redux state, `videoTranformedView` (maybe).

Reply:
I renamed this style class, thanks.

Not sure however what you mean here exactly. I used the word "transform(s)" everywhere in the VideoTransform since the name of the component predefines that it's all about video. On the other hand I couldn't imagine a situation where we (visually) transform anything other than a video stream. Or did I maybe misunderstand your comment?

Reply:
There's "transforms" in the redux state of "base/media". But "base/media" contains both audio and video-related state. So "transforms" is unclear to me.

Reply:
The transforms object is in the video subtree of the state of the feature, as it's being reduced by the video reducer that is later combined with the audio reducer to handle the base/media tree.